base/experiments/ad9910_ram.py

```diff
@@ -101,13 +103,15 @@ class RAMProfile:
             - RAM Bidirectional Ramp Mode (p.38, Figure 46)
             - RAM Continuous Bidirectional Ramp Mode (p.39, Figure 47)
             - RAM Continuous Recirculate Mode (p.40, Figure 48)
+        dwell_end: negated no-dwell high bit (default: 1). Ignored except for in
+        RAM Ramp-Up Mode.
 
     Raises:
         ValueError: Unsupported RAM configuration found.
     """
     RAM_SIZE = 1024
 
-    def __init__(self, dds, data, ramp_interval, ram_type, ram_mode):
+    def __init__(self, dds, data, ramp_interval, ram_type, ram_mode, dwell_end=1):
```

Comment:
I prefer the user-facing API to use the Boolean type instead of 0/1 binary.
```suggestion
    def __init__(self, dds, data, ramp_interval, ram_type, ram_mode, dwell_end=True):
```

Reply:
Good -- had forgotten that I convert to int in `__init__()` before passing `nodwell_high` to `set_profile_ram`, so no need to use 0/1 for user input